examples/subt/mytimer.py

```diff
@@ -0,0 +1,30 @@
+"""
+  Timer
+"""
+import math
+from threading import Thread
+
+from osgar.bus import BusShutdownException
+
+
+class MyTimer(Thread):
```

Comment:
Why are all the nodes using Thread as a base class? I was under the impression that Node is the new base class.

Reply:
I guess that these classes were written before `Node` was ready and merged in master. You are right that it could/should be refactored now.